roles/balloonprinter/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+balloon_version: 2.5.783 # the version from github icpctools releases
```

Comment:
Why do we hardcode the version?

Reply:
Because the latest version wasn't functional (something was/is broken in the build process so that the zips aren't produced/attached to the GitHub release under some circumstances)

Reply:
There was a problem with the icptools latest build from building/publishing.